src/apps/btc/btc_sign.c

```diff
@@ -44,6 +46,8 @@ static uint32_t _num_inputs;
 static uint32_t _num_outputs;
 static uint32_t _locktime;
 static uint32_t _index;
+static bool _rbf;
```

Comment:
i would also guard this with APP_BTC, so it's clear that it is not handled in LTC.